extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -468,7 +480,10 @@ export class SafeframeHostApi {
     if ((payload['push'] && !this.expandByPush_) ||
```

Comment:
This is hard to parse.  Could we break it into separate checks or at least add some comments?